ArcadiaParties-SPA/src/app/user/effects/effect.ts

```diff
@@ -11,7 +11,10 @@ export class UserEffect {
   loadUser = createEffect(() =>
```

Comment:
7-8 empty line. Can't highlight, cause they were here before this PR

Reply:
It deleted